tools/deb/BUILD

```diff
@@ -0,0 +1,52 @@
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
+
+pkg_tar(
+    name = "envoy-bin",
+    files = [
+        "istio-iptables.sh",
+        "istio-start.sh",
+        "//src/envoy/mixer:envoy",
+    ],
+    mode = "0755",
+    package_dir = "/usr/local/bin",
+)
+
+pkg_tar(
+    name = "istio-conf",
+    files = [
+        "envoy.json",
+        "sidecar.env",
+    ],
+    mode = "0755",
+    package_dir = "/var/lib/istio/envoy",
+)
+
+pkg_tar(
+    name = "istio-systemd",
+    files = ["istio.service"],
+    mode = "644",
+    package_dir = "/lib/systemd/system",
+)
+
+pkg_tar(
+    name = "debian-data",
+    extension = "tar.gz",
+    deps = [
+        ":envoy-bin",
+        ":istio-conf",
+        ":istio-systemd",
+    ],
+)
+
+pkg_deb(
+    name = "istio-proxy",
```

Comment:
can we call this envoy-istio ? That is more truthful and would be in line with the terminology we use everywhere in docs. We try to avoid istio proxy in docs.

Reply:
https://istio.io/search/?q=%22istio%20proxy%22 has more hits than "envoy istio" or "istio envoy"
the repo is called proxy, may eventually have more than 1 proxy implementation, I don't think it needs to be named envoy here  (maybe sidecar is better than proxy)

Reply:
istio-sidecar-envoy. For precisely the same reasons as @ldemailly stated above, we are packaging Envoy (not nginx or linkerd). So, it needs to be qualified. Don't care if its istio-prox-envoy or istio-envoy or whatever.. But please do use envoy in the name. 

And FYI, from the style guide
```Envoy

We prefer to use “Envoy” as it’s a more concrete term than “proxy” and will resonate if used consistently throughout the docs.

Synonyms:

“Envoy sidecar” - ok
“Envoy proxy” - ok
“The Istio proxy” – best to avoid unless you’re talking about advanced scenarios where another proxy might be used.
“Sidecar” – mostly restricted to conceptual docs
“Proxy – only if context is obvious
```